beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -715,6 +715,10 @@ impl<T: BeaconChainTypes> Worker<T> {
             &metrics::BEACON_BLOCK_GOSSIP_SLOT_START_DELAY_TIME,
             block_delay,
         );
+        metrics::set_gauge(
+            &metrics::BEACON_BLOCK_LAST_DELAY,
```

Comment:
We could miss an update to this gauge if we don't poll at the right time, right? Maybe a debug log with the number in milliseconds would be better?

Although I guess the really late ones will get picked up here:

https://github.com/sigp/lighthouse/blob/38514c07f222ff7783834c48cf5c0a6ee7f346d0/beacon_node/network/src/beacon_processor/worker/gossip_methods.rs#L744-L751

Reply:
Yep. I've used a metric like this in the recent past and it has proven to be more effective than the current version of doing like an average of the histogram sum and bucket numbers. 

It may not be perfectly accurate because the resolution is now the time if the query and we lose blocks delays based on the query time, but having this metric over not having it, is strictly an improvement imo.

Reply:
Yeah, buckets are annoying. Let's do it.